setup_pipeline.sh

```diff
@@ -0,0 +1,6 @@
+oc create -f https://raw.githubusercontent.com/raynalee4/mlperf-tekton/master/pipeline/resources.yml
+oc create -f https://raw.githubusercontent.com/raynalee4/mlperf-tekton/master/task/buildah.yaml
+oc create -f https://raw.githubusercontent.com/raynalee4/mlperf-tekton/master/task/run.yml
+oc create -f https://raw.githubusercontent.com/raynalee4/mlperf-tekton/master/pipeline/pipeline.yml
+oc create -f https://raw.githubusercontent.com/raynalee4/mlperf-tekton/master/pipeline/pipelinerun.yml
```

Comment:
Kindly create 2 scripts. One for setting up tekton resources. And another for running different pipelineruns.